learning_journal/views.py

```diff
@@ -0,0 +1,72 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound
+from .form import JournalForm
+from sqlalchemy.exc import DBAPIError
+from sqlalchemy import desc
+import transaction
+import markdown
+
+from .models import (
+    DBSession,
+    Entry,
+    )
+
+
+@view_config(route_name='home', renderer='templates/list_view.jinja2')
+def list_view(request):
+    """Handle the view of our home page."""
+    return {'entries': DBSession.query(Entry).order_by(desc(Entry.created)).all()}
```

Comment:
good.  Sometimes I find it useful to assign my query to a temporary local variable simply for debugging purposes.  But this code is exactly right.